swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md

```diff
@@ -1,67 +1,137 @@
+
+For example, consider a scenario in which the configurable variables are defined in the following way,
+
+```ballerina
+configurable int port = 9090;
+configurable float maxPayload = ?;
+configurable string username = ?;
+configurable boolean verbose = true
+```
+
+and the values are provided in two `Config.toml` and `info.toml` files as follows.
+
+**Config.toml**
+
+```toml
+port = 9000
+username = "admin-user"
+```
+
+**info.toml**
+
+```toml
+maxPayload = 1.0
+verbose = true
+```
+
+Accordingly, execute the commands below to provide the values via the `BAL_CONFIG_FILES` environment variable based on your operating system.
+
+**For Windows:**
+
+```
+$ set BAL_CONFIG_FILES=<path-to-info.toml>;<path-to-Config.toml>
+```
+
+**For Linux/macOS:**
+
+```
+$ export BAL_CONFIG_FILES=<path-to-info.toml>:<path-to-Config.toml>
+```
+
+## Provide via environment variables
+
+You can also provide configuration values through an environment variable named `BAL_CONFIG_DATA` in which the content is expected to comply with the [TOML syntax](#toml-syntax). 
+
+For example, consider a scenario in which the configurable variables are defined in the following way,
+
+```ballerina
+configurable int port = 9000;
+configurable float maxPayload = ?;
+configurable string username = ?;
+configurable boolean verbose = ?;
+```
+
+Accordingly, execute the commands below to configure the values via an environment variable based on your operating system.
+
+**For Windows:**
+
+```
+$ set BAL_CONFIG_DATA=maxPayload=1.0\nusername="user1"\nverbose=true
+
+```
+
+**For Linux/macOS:**
+
+```
+$ export BAL_CONFIG_DATA='maxPayload=1.0\nusername="user1"\nverbose = true'
+```
+
+## TOML syntax
 
-### Provide via TOML syntax
+Ballerina defines a specific TOML syntax based on the <a href="https://toml.io/en/v0.4.0" target="_blank">TOML(v0.4) format</a> to be used when configuring the variables through the configuration files and environment variables. 
 
-Ballerina defines a specific TOML syntax to be used when configuring the variables through the configuration files and
-environment variables. Depending on the type of the configurable variable, the way of providing values in the TOML
-content differs. Currently, TOML-based configuration is supported for configurable variables of `int`, `float`,
-`boolean`, `string`, `xml`,  `decimal`, `enum`, the arrays of the respective types, map, record, table and the union of
-the respective types.
+>**Info:** The way of providing values in the TOML content differs depending on the type of the configurable variable. Currently, the TOML-based configuration supports configurable variables of types `int`, `float`, `boolean`, `string`, `xml`,  `decimal`, `enum`, the arrays of the respective types, `map`, `record`, `table`, and the union of the respective types.
 
-The mapping of Ballerina types to TOML types can be explained through the following examples:
+The examples below explain the mapping of Ballerina types to TOML types.
```

Comment:
All the Ballerina records within the table are not properly indented.

Reply:
Fixed.